src/main/java/ch/njol/skript/hooks/economy/classes/Money.java

```diff
@@ -22,7 +22,7 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 import ch.njol.skript.Skript;
-import ch.njol.skript.classes.Arithmetic;
+import org.skriptlang.skript.lang.arithmetic.Arithmetic;
```

Comment:
Sorry I'm a little confused with this file now, is there any reason for this import now? From what I can tell we don't use Arthmetic in this file now